packages/valory/skills/tx_settlement_multiplexer_abci/rounds.py

```diff
@@ -139,6 +144,10 @@ class FinishedBetPlacementTxRound(DegenerateRound):
     """Finished bet placement round."""
 
 
+class FinishedSellOutcomeTokensTxRound(DegenerateRound):
+    """Finished sell outcome tokens round."""
+
+
```

Comment:
Not utilized by the `TxSettlementMultiplexerAbciApp`. Is this going to be part of another PR?